manifests/klusterlet/managed/klusterlet-work-clusterrole.yaml

```diff
@@ -33,3 +33,7 @@ rules:
 - apiGroups: ["work.open-cluster-management.io"]
   resources: ["appliedmanifestworks/finalizers"]
   verbs: ["update"]
+# Allow agent to operate leases
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
```

Comment:
these are agent rbac on hub I think, lease is not needed

Reply:
his is the agent clusterrole on spoke, we already add the lease permission to the spoke agent role (klusterlet/management/klusterlet-work-role.yaml) , so rollbacked